License2Deploy/rolling_deploy.py

```diff
@@ -250,6 +252,30 @@ def revert_deployment(self): #pragma: no cover
     logging.error("REVERT COMPLETE!")
     exit(self.exit_error_code)
 
+  def disable_all_cloudwatch_Alarms(self):
+    ''' Disable all the cloud watch alarms '''
+    cloud_watch_alarms = filter(lambda alarm: self.project in alarm.name and self.env in alarm.name, self.conn_cloudwatch.describe_alarms()) 
+    for alarm in cloud_watch_alarms:
+      logging.info("Found an alarm. {0}".format(alarm.name))
+      alarm_actions_disabled = self.conn_cloudwatch.disable_alarm_actions(alarm.name)
+      if alarm_actions_disabled :
```

Comment:
if alarm_actions_disabled: << no space between alarm_actions_disabled and the colon please